app/frontend/src/lib/components/SimpleFile/Component.js

```diff
@@ -36,7 +36,7 @@ import editForm from './Component.form';
 import { Constants } from '../Common/Constants';
 var uniqueName = Utils.uniqueName;
 var ID = 'simplefile';
-var DISPLAY = 'Upload';
+var DISPLAY = 'File Upload';
```

Comment:
We should not be directly editing the lib/components files ad this is directly generated by the Typescript compiler from the top-level components folder. It should be changed there instead and then recompiled... but there's other build mismatch issues associated with that atm.

Reply:
Yeah would figure out the rebuild process soon for stuff like this. Again maybe move out of scope for this fix.